pkg/download/mode/mode.go

```diff
@@ -72,12 +104,11 @@ func NewFile(m Mode, downloadURL string) (*DownloadFile, error) {
 		return parseFile(bts)
 	}
 
-	switch m {
-	case Sync, Async, Redirect, AsyncRedirect, None:
-		return &DownloadFile{Mode: m, DownloadURL: downloadURL}, nil
-	default:
-		return nil, errors.E(op, errors.KindBadRequest, fmt.Sprintf(invalidModeErr, m))
+	retFile := &DownloadFile{Mode: m, DownloadURL: downloadURL}
```

Comment:
Just a nit: what does retFile stand for? maybe we can just call it `df` for DownloadFile

Reply:
a heavily abbreviated "file to return". I see `df` in a lot of other places, so happy to change it to that.